Sources/ApolloCodegenLib/ApolloCodegenOptions.swift

```diff
@@ -93,7 +94,9 @@ public struct ApolloCodegenOptions {
   ///  - suppressSwiftMultilineStringLiterals: Don't use multi-line string literals when generating code. Defaults to false.
   ///  - urlToSchemaFile: The URL to your schema file.
   ///  - downloadTimeout: The maximum time to wait before indicating that the download timed out, in seconds. Defaults to 30 seconds.
+  ///  - additionalInflectionRules: Any non-default rules for pluralization or singularization you wish to include. Defaults to an empty array. Only used by the Swift code generation engine.
```

Comment:
I feel like parameters should be ordered in documentation the same order as they are in the function. So if this is the second parameter, it should be second in the docs too.

Reply:
Not a problem - done in https://github.com/apollographql/apollo-ios/pull/1879/commits/d6413e203ab8e851904a7bf7e3d6826073f2685c.